pages/api/tokens/index.ts

```diff
@@ -1,20 +1,28 @@
-import {withCors} from "middleware";
 import {NextApiRequest, NextApiResponse} from "next";
-import { Op, Sequelize } from "sequelize";
+import { Op, Sequelize, WhereOptions } from "sequelize";
 
 import Database from "db/models";
 
+import {withCors} from "middleware";
+
 import { error as logError } from 'services/logging';
 
 const colToLower = (colName: string) => Sequelize.fn("LOWER", Sequelize.col(colName));
 
 async function get(req: NextApiRequest, res: NextApiResponse) {
-  const { networkName } = req.query;
+  const { networkName, chainId } = req.query;
```

Comment:
Just noticed that we have a `pages/api/search/tokens` that looks _a lot_ like this, maybe we can use that one instead and just join the two